scripts/samplesheet_generator.py

```diff
@@ -294,6 +294,56 @@ def gen_Miseq_data(pro):
     return ("{}{}".format(header, str_data), data)
 
 
+def gen_Nextseq_lane_data(pro):
+    data=[]
+    header_ar = ["FCID","Lane","Sample_ID","Sample_Name","Sample_Ref","index","index2","Description","Control","Recipe","Operator","Sample_Project"]
+    for out in pro.all_outputs():
+        if  out.type != "Analyte":
+            continue
+        for sample in out.samples:
+            sp_obj = {}
+            sp_obj['lane'] = out.location[1].split(':')[0].replace(',','')
+            sp_obj['sid'] = "Sample_{}".format(sample.name).replace(',','')
+            sp_obj['sn'] = sample.name.replace(',','')
+            try:
+                sp_obj['pj'] = sample.project.name.replace('.','__').replace(',','')
+            except:
+                #control samples have no project
+                continue
```

Comment:
There are very few comments in this file that explains what's going on (5 comments for ~500 lines of code 😅but that's not your fault), here is one of the few but I still don't think it explains the:
Maybe you should specify which type of error you're catching e.g. `KeyError` or `AttributeError`. Is it safe to assume a missing `sample.project.name` will always be a "control sample" and not due to some other error?

Reply:
Thank @jgruselius! The exception handling may take a while to fix because I need to understand which type of error it can occur. We will definitely inplement it for future improvement.